### PR TITLE
Branch view client

### LINKED ISSUE #81: View appointment command

### PR DESCRIPTION
**Pull request by JumpyJay**
Fixes #81

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 1]
```diff
@@ -16,15 +16,19 @@ public class CommandResult {
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+    /** Client information should be shown to the user. */
+    private final boolean showClient;
+
     /** The application should exit. */
     private final boolean exit;
```
Comment (jasmiinee) [inline src/main/java/seedu/address/logic/commands/CommandResult.java:23]: Should these boolean variables be named as `isExit` , `isShowHelp`, `isShowClient` instead?

File: src/main/java/seedu/address/logic/commands/ViewClientCommand.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private final Name clientName;
+
+    public ViewClientCommand(Name clientName) {
+        this.clientName = clientName;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        List<Person> clientList = model.getFilteredPersonList();
+        if (clientList.stream().noneMatch(person -> person.getName().equals(this.clientName))) {
```
Comment (sarahteonin) [inline src/main/java/seedu/address/logic/commands/ViewClientCommand.java:32]: Could this expression be abstracted to be clearer?

**Review by sarahteonin — COMMENTED** [review]
Overall LGTM!

File: src/main/resources/view/ViewWindow.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    -fx-text-fill: black;
+}
+
+#viewClientContainer {
+    -fx-background-color: #D3EDEA;
+}
+
+#clientProfileContainer {
+    -fx-background-color: #009794;
+}
```
Comment (jiayr99) [inline src/main/resources/view/ViewWindow.css:11]: LGTM!

File: src/main/java/seedu/address/ui/ClientWindow.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+     *         <li>
+     *             if {@code dialogStage} is already showing.
+     *         </li>
+     *     </ul>
+     */
+    public void show() {
+        logger.fine("Showing information page about the client.");
+        getRoot().show();
+        getRoot().centerOnScreen();
+    }
```
Comment (GuanpengR) [inline src/main/java/seedu/address/ui/ClientWindow.java:103]: Nice that the popup() is set to center of screen to make it more prominent!

File: src/main/resources/view/ClientWindow.fxml
Comment (GuanpengR) [inline src/main/resources/view/ClientWindow.fxml:1]: LGTM!

**Pull request merged**